commercetools-models/src/main/java/io/sphere/sdk/discountcodes/commands/updateactions/SetCustomType.java

```diff
@@ -0,0 +1,45 @@
+package io.sphere.sdk.discountcodes.commands.updateactions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.discountcodes.DiscountCode;
+import io.sphere.sdk.types.customupdateactions.SetCustomTypeBase;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+/**
+ *
+ * Sets/unsets the custom type.
+ *
+ * {@doc.gen intro}
+ *
+ * {@include.example io.sphere.sdk.discountcodes.DiscountCodeCustomFieldsIntegrationTest#setCustomType()}
+ *
+ */
+public final class SetCustomType extends SetCustomTypeBase<DiscountCode> {
```

Comment:
Same for `SetCustomTypeBase`, wouldn't it be more convenient to add `ofTypeKeyAndObjects `, `ofTypeIdAndObjects`, `ofTypeIdAndJson `, `ofTypeKeyAndJson ` and `ofRemoveType` to `SetCustomTypeBase` ?